### PR TITLE
Add WordLift Loader

### PR DESCRIPTION
**Pull request by msftwarelab**

### CODE REVIEW

File: llama_hub/wordlift/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Here's an example of how to use the WordLift GraphQL Reader:
+
+```python
+import json
+import logging
+from llama_index import GPTVectorStoreIndex
+from llama_index.readers.schema.base import Document
+from langchain.llms import OpenAI
+from langchain.chains.question_answering import load_qa_chain
+from base import WordLiftGraphQLReader
```
Comment (jerryjliu) [inline llama_hub/wordlift/README.md:22]: would replace this with `from llama_hub.wordlift.base import WordLiftGraphQLReader`

File: llama_hub/wordlift/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+2. Create an instance of the `WordLiftGraphQLReader` class, passing in the configuration options.
+3. Use the `load_data` method to fetch and transform the data.
+4. Process the loaded documents as needed.
+
+Here's an example of how to use the WordLift GraphQL Reader:
+
+```python
+import json
+import logging
+from llama_index import GPTVectorStoreIndex
```
Comment (jerryjliu) [inline llama_hub/wordlift/README.md:18]: should now be `VectorStoreIndex`, the `GPT` prefix is deprecated

**Review by jerryjliu — COMMENTED** [review]
nice work! some minor comments:

- let's not check in `poetry.lock`

File: llama_hub/wordlift/README.md (hunk 1 above)
Comment (msftwarelab) [reply llama_hub/wordlift/README.md]: Resovled

File: llama_hub/wordlift/README.md (hunk 2 above)
Comment (msftwarelab) [reply llama_hub/wordlift/README.md]: Resolved

**Review by jerryjliu — CHANGES_REQUESTED** [review]
@msftwarelab bump on removing poetry.lock as a dep (everything else looks good)

File: llama_hub/wordlift/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+langchain
```
Comment (jerryjliu) [inline llama_hub/wordlift/requirements.txt:1]: you don't need this, openai, or llama-index (all of these are installed as part of the core llama-index package which is already a dep for llama_hub)

Comment (msftwarelab) [reply llama_hub/wordlift/requirements.txt:1]: @jerryjliu Resolved

**Review by jerryjliu — APPROVED** [review]

File: llama_hub/library.json [hunk 4]
```diff
@@ -252,6 +252,11 @@
     "author": "sidu",
     "keywords": ["jokes", "dad jokes"]
   },
+  "WordLiftLoader": {
+    "id": "WordLift",
```
Comment (jerryjliu) [inline llama_hub/library.json:256]: nit: should be same as folder name. can you change this to `"id": "wordlift"`? thanks

Comment (msftwarelab) [reply llama_hub/library.json]: @jerryjliu I changed to `"id": "wordlift"`. Thanks.

**Pull request merged**